common/src/androidTest/kotlin/DatabaseTest.kt

```diff
@@ -0,0 +1,83 @@
+import com.cuhacking.atlas.db.*
+import com.cuhacking.atlas.db.Database.Companion.Schema
+import com.squareup.sqldelight.db.SqlDriver
+import com.squareup.sqldelight.sqlite.driver.JdbcSqliteDriver
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import kotlin.test.assertTrue
+
+lateinit var driver: SqlDriver
+lateinit var queries: FeatureQueries
+
+class DatabaseTest {
+
+    private val sampleFeature = GeoJsonAdapter.decode("""{"type":"Feature","geometry":{"type":"Point","coordinates":[-75.0,45.0]},"properties":{}}""")
+
+    @Before
+    fun setup() {
+        driver = JdbcSqliteDriver(JdbcSqliteDriver.IN_MEMORY)
+        Schema.create(driver)
+        val database = driver.createDatabase()
+        queries = database.featureQueries
+
+        val feature1 = Feature(1, "River Building","Richcraft Hall","Building",null,null,null,sampleFeature)
```

Comment:
My advice would be to take these (and any other) features and declare them outside of this function.
Then, add a helper extension function that converts a `Feature` into its equivalent `Feature_fts`. 

This way you have a set of features that you can use anywhere, and then the helper function will make it more reusable.